sonar/modules/documents/api.py

```diff
@@ -40,13 +50,13 @@ class DocumentRecord(SonarRecord):
     @classmethod
     def create(cls, data, id_=None, **kwargs):
```

Comment:
remove this method

Reply:
This method is needed for institution resolution in [documents import CLI](https://github.com/rero/sonar/pull/21/files#diff-5288bcb3dcedc9ff453c3369a6c20f6bR95), so it can't be removed